### PR TITLE
Pricing docs

### PR DESCRIPTION
**Pull request by matmair**
More information on the pricing view and other changes of https://github.com/inventree/InvenTree/issues/1656.

### CODE REVIEW

**Comment by matmair** [comment]
@SchrodingersGat @eeintech mind checking ([build version](https://inventree--146.org.readthedocs.build/en/146/part/pricing/)) this first attempt at documenting pricing? I did not contain screenshots as those tend to be old fairly fast right now.
For explaining the graphs some might be needed in the future.

**Comment by SchrodingersGat** [comment]
I like this addition - any thoughts @eeintech ?

File: docs/part/pricing.md [hunk 1]
```diff
@@ -6,7 +6,59 @@ title: Pricing
...
+A price range describes the unit price for a part at different buying / selling quantities (one par is called price point following). Many suppliers sell their parts with different price points depending on packaging and quantities. That can be replicated in InvenTree with price ranges.
+
+The simplest price range just contains one price point - this will be created when a part gets assigned a price in a form.
+
+!!! warning "Adjust the wanted amount!"
+    As parts can have price ranges it is important to set the correct amount in the first panel to get an accurate price.
+
+### Currency in the graphs and tables
+
+All money-values are adjusted to the default currency at the current exchange rates (see [currency conversion](#/#currency-conversion) for more information). This can distort historical data but is necessary to easily compare values.
```
Comment (eeintech) [inline docs/part/pricing.md:40]: Shouldn't it be `#currency-conversion` (no `#/`)?

**Comment by eeintech** [comment]
> The rates can also be set manually in the settings. 

I don't think it is the case anymore, is it?

Other than that, it looks great!

File: docs/part/pricing.md (hunk 1 above)
Comment (matmair) [reply docs/part/pricing.md:40]: Yes you are right about that, thanks.

**Comment by matmair** [comment]
> 
> 
> > The rates can also be set manually in the settings.
> 
> I don't think it is the case anymore, is it?
> 
> Other than that, it looks great!

Ah yes, I think we should bring that back. I loved it for debugging ;-) .

**Comment by matmair** [comment]
@SchrodingersGat this would be ready for merge now

**Pull request merged**